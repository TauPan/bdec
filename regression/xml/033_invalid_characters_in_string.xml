<protocol>
    <field name="a" length="9 * 8" type="text" />

</protocol>
