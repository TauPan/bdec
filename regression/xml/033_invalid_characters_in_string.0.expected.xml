<a>1?2?3&lt;4&gt;5</a>